python/paddle/fluid/tests/unittests/test_listen_and_serv_op.py

```diff
@@ -71,16 +71,17 @@ def _start_pserver(self, use_cuda, sync_mode):
 
     def _wait_ps_ready(self, pid):
         retry_times = self.ps_timeout
+        sleep_time = 0.5
         while True:
             assert retry_times >= 0, "wait ps ready failed"
-            time.sleep(0.5)
+            time.sleep(sleep_time)
             try:
                 # the listen_and_serv_op would touch a file which contains the listen port
                 # on the /tmp directory until it was ready to process all the RPC call.
                 os.stat("/tmp/paddle.%d.port" % pid)
                 return
             except os.error:
-                retry_times -= 1
+                retry_times -= sleep_time
```

Comment:
This change seems is not for current PR? and `retry_times` seems is the total count for trying not the time.

Reply:
I changed the name ```retry_times``` to ```start_left_time``` to indicate that this is the left time for pserver starting, and this change is for passing the CI